assets/src/stories-editor/helpers/useMoveBlockToPage.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * WordPress dependencies
+ */
+import { useDispatch, useSelect } from '@wordpress/data';
+import { cloneBlock } from '@wordpress/blocks';
+
+/**
+ * Hook that exposes functions relevant for moving an element to a neigboring page.
+ *
+ * @param {string} blockId  Id of block to move
+ * @return {Object} Returns two functions: `getPageByOffset` and `moveBlockToPage`.
+ */
+const useMoveBlockToPage = ( blockId ) => {
+	const pages = useSelect( ( select ) => select( 'core/block-editor' ).getBlockOrder() );
+	const block = useSelect( ( select ) => select( 'core/block-editor' ).getBlock( blockId ) );
+	const currentPageId = useSelect( ( select ) => select( 'amp/story' ).getCurrentPage() );
```

Comment:
I think it would be clearer to do 
```
const {
		pages,
		block,
		currentPageId,
} = useSelect(
		( select ) => {
			const {
				getBlockOrder,
				getBlock,
			} = select( 'core/block-editor' );
			const { getCurrentPage } = select( 'amp/story' );
			return {
				pages: getBlockOrder(),
	                        block: getBlock( blockId ),
		                currentPageId: getCurrentPage()
			};
		}, [ blockId ]
      );
```

Reply:
See example from https://github.com/ampproject/amp-wp/blob/8ae3c05f0f525ed85305527777d5ef07d40a3ae0/assets/src/stories-editor/blocks/amp-story-page/copy-paste-handler.js#L19-L37

Reply:
Clearer how? I don't think it's any more clear.

I will add the dependencies to the `useSelect`'s though - forgot it took those.

But I actually don't think that the more elaborate single `useSelect` is an improvement. With the dependencies it will actually also be less optimal, as two of the variables don't need to update just because `clientId` does.

Reply:
It is easier to read as you don't repeat `useSelect( ( select ) => select(`. Long lines of code are harder to scan. That is why it was done that way in the other code snippet I sent over. 

It is just also how I have seen hooks used elsewhere in other projects. I don't feel super strongly about this one, functionality the code is fine, just don't like the style of it.

Reply:
Added dependencies in 3241e3b.

Reply:
As I don't feel super strongly about this, I am going to mark this as resolved.